src/components/CompareResults/RevisionRow.tsx

```diff
@@ -248,6 +266,15 @@ function determineSign(baseMedianValue: number, newMedianValue: number) {
   return '';
 }
 
+function determineStandardDeviationSign(
+  baseStandardDeviation: number,
+  newStandardDeviation: number,
+) {
+  if (baseStandardDeviation > newStandardDeviation) return '>';
+  if (baseStandardDeviation < newStandardDeviation) return '<';
+  return '';
+}
+
```

Comment:
Maybe we should move this function to the [utils/helper.tsx](https://github.com/mozilla/perfcompare/blob/main/src/utils/helpers.ts) file as it is being used more than once.

Reply:
yes did that but I think there are many methods which could be in helper classes as they are just repeating.